### PR TITLE
Update mathpix-snipping-tool to latest

### PR DESCRIPTION
**Pull request by jinntrance**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/mathpix-snipping-tool.rb [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+  sha256 :no_check
 
-  url "https://mathpix.com/dmg/SnippingTool-v#{version}.dmg"
-  appcast 'https://mathpix.com/appcast.xml'
+  url 'https://mathpix.com/dmg/snip.dmg'
   name 'Mathpix Snipping Tool'
   homepage 'https://mathpix.com/'
 
-  auto_updates true
+  auto_updates false
```
Comment (reitermarkus) [inline Casks/mathpix-snipping-tool.rb]: Remove this line, `false` is the default.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

**Comment by reitermarkus** [comment]
1.3.1 still seems to be the latest release, so why remove the appcast?

**Comment by jinntrance** [comment]
The download url from the official site has changed, and version is no longer included.

**Comment by vitorgalvao** [comment]
1. The `appcast` is a Sparkle appcast, still present in the app. I doubt this no longer auto updates.
2. Maybe the URL on the website has changed, but the appcast (which is clearly official) still has versioned links, and the unversioned link downloads the same version.

Everything in this cask appears valid as is.

File: Casks/mathpix-snipping-tool.rb (hunk 1 above)
Comment (zbeekman) [reply Casks/mathpix-snipping-tool.rb]: LGTM (now)

**Comment by zbeekman** [comment]
I think this can ship, no?

**Comment by vitorgalvao** [comment]
@zbeekman Not really, no. None of the concerns were addressed. As it is, this is closer to being closed than being merged.

**Comment by zbeekman** [comment]
> Everything in this cask appears valid as is.

Sorry, I completely misunderstood your comment.

**Pull request closed without merging**